pkgs/development/ruby-modules/rbenv/default.nix

```diff
@@ -0,0 +1,42 @@
+{stdenv, fetchFromGitHub, bash}:
+stdenv.mkDerivation {
+  name = "rbenv";
+  
+  # fetchFromGitHub is a build support function that fetches a GitHub
+  # repository and extracts into a directory; so we can use it
+  # fetchFromGithub is actually a derivation itself :)
+  src = fetchFromGitHub {
+    owner = "rbenv";
+    repo = "rbenv";
+    rev = "v1.1.2";
+    sha256 = "12i050vs35iiblxga43zrj7xwbaisv3mq55y9ikagkr8pj1vmq53";
+  };
+
+  buildPhase = ''
+  	${bash}/bin/bash src/configure
+  	make -C src
+  '';
```

Comment:
How about this?

```suggestion
  preConfigure = "cd src";
```

Reply:
Not overriding standard build phases usually is better. In this case it would for example fix cross compiling.

Reply:
Maybe also needs `bash` in `nativeBuildInputs`...

Reply:
so by changing directory in `postPatch`; the typical `buildPhase` will work ?